tcms/testcases/tests/test_views.py

```diff
@@ -564,3 +566,33 @@ def test_disabled_priority_now_shown(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertContains(response, 'Set P3')
         self.assertNotContains(response, 'Set P4')
+
+
+class TestShowCases(BasePlanCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.request = RequestFactory()
+
+    def test_get_selected_testcases_works_with_string_pks(self):
+        """
+        Assures that tcms.testcases.views.get_selected_testcases
+        returns the same results, regardless of whether the
+        passed request contains the case pks as strings or
+        integers, as long as they are the same pks.
+        """
+
+        case_int_pks = [1, 2, 3, 4, 5, 6]
```

Comment:
Hmmm, where are these IDs coming from ? You should use existing variables (or create new ones if needed) and their primary keys, not hard-coded values. Check-out what data exists in your base class.